controller/src/test/java/io/pravega/controller/store/stream/StreamMetadataStoreTest.java

```diff
@@ -72,7 +72,8 @@
 
     //Ensure each test completes within 10 seconds.
     @Rule
-    public Timeout globalTimeout = new Timeout(30, TimeUnit.SECONDS);
+    // TODO: shivesh
```

Comment:
that was on an outdated change set (i had already removed that todo).
i add todos when i increase test timeouts for testing under debugger and clean them up after completing the testing.. this may have slipped into a commit.